libs/location/src/routes/locations.ts

```diff
@@ -0,0 +1,23 @@
+import type { Request, Response } from "express";
+import { getAllLocations } from "../repository/queries.js";
+import { searchLocations } from "../repository/service.js";
+
+export const GET = async (req: Request, res: Response) => {
```

Comment:
Any idea what is this file for? It is under routes directly.

Reply:
This is a core API endpoint that makes the location autocomplete feature work!